src/sap.ui.core/src/sap/ui/qunit/qunit-coverage.js

```diff
@@ -64,7 +64,9 @@
 
 				if (jQuery && jQuery.sap) {
 					jQuery.sap.require._hook = function(sScript, sModuleName) {
-						// TODO: manage includes/excludes? (usage of regex)
+						// manage includes and excludes with blanket utils
+						var bFiltered = window.blanket.utils.matchPatternAttribute(sModuleName, window.blanket.options("filter"));
+						bFiltered &= !window.blanket.utils.matchPatternAttribute(sModuleName, window.blanket.options("anti-filter"));
```

Comment:
According to the implementation in blanket.js the options would be `filter` and `antifilter` (without the `-`).
See https://github.com/alex-seville/blanket/blob/523ff3bbd3ce9f7f9e3139ccef71d20bac314a9b/src/config.js#L8-L13).

Furthermore I wonder about the evaluation order. I would prefer to do it as in blanket.js (exclude first):
https://github.com/alex-seville/blanket/blob/49eadb27720efd40a9c1ce14c592c25d3a405780/dist/qunit/blanket.js#L5070-L5079